linux/vhba_hot_add_remove/freebsd_create_disk_partition.yml

```diff
@@ -63,21 +79,9 @@
         guest_disk_list_before_reboot: []
         guest_disk_list_after_reboot: []
```

Comment:
The block name is not proper. And here doesn't need to be so complicated. For 32bit FreeBSD, you can just:
1. Reboot
2. Get the partition info by uuid
3. Get the partition name and device path from partition info.

which will be less time-consuming than your current steps.